style.css

```diff
@@ -458,3 +470,121 @@
 hr {
   width: 100%;
 }
+
+.contact-form {
+  margin-top: 150px;
+  border-top-left-radius: 68px;
+  display: flex;
+  flex-direction: column;
+  width: 373px;
+  height: 800px;
+  background: #6070ff;
+  background-image: url("images/contact\ form.png");
+  background-size: cover;
+  background-repeat: repeat-x;
+}
+
```

Comment:
- [ ] You did a great job 😊 at adding the background image, but kindly as stated by the previous reviewer try to put the image in the correct position on the contact form, and with that, it will be like the Figma Design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/220426344-0a6f26f1-d386-49ff-a9fa-cdca55526404.png)     |   ![image](https://user-images.githubusercontent.com/83360636/220427457-13d15399-78e7-458f-b461-cc4a63a0d81e.png)  |